SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -51,6 +51,8 @@ You can find out more about the Console in the [Glossary]({{urlRoot}}/content/gl
 
 **Note:** You must close the Unreal Editor before building your workers. If the Editor is open when you try to build your workers the command will fail.
 
+**Note:** By default, Spatial networking is enabled for Unreal GDK projects. However, if the `bSpatialNetworking` option is visible in your `DefaultGame.ini` configuration file (located in `<ProjectRoot>\Game\Config` directory), ensure that it is set to `True` to enable Spatial networking for your cloud deployment.
```

Comment:
New people could come across it when going through tutorial if they play around with `Play` options in the editor and leave `Spatial Networking` to false before closing the editor. 

Regarding the points raised:

1: Would "networking using Spatial" sound clearer than "Spatial networking"? Or would it be better to link this to one of the Spatial OS concepts docs? I attempted to clarify it, but not sure it made it much clearer.

2: it's one of the Unreal configuration files. Are we fine assuming background knowledge of Unreal? Even if not, I'm not sure how we can make it clearer - it's a configuration file in the specified location.

3 and 4: Tried to clarify it, once again, not sure this is much clearer.

Reply:
This bit is great!
`By default, Spatial networking is enabled for Unreal GDK projects.`
Super helpful.

Because it is an early tutorial, we can hand-hold a little more. It's not entirely clear why I should be looking in the file or if I need to do anything.

Also: question: What if the `SpatialNetworking` option is not visible. And by "visible", do we mean, it's in the file?

Also, if there are several notes, we tend to have to make it easier to follow.
```
**Notes:**

* Note here about something.
* Another note here about something.
```

```suggestion
**Note:** You might need to reset Spatial networking.</br>
By default, Spatial networking is enabled for Unreal GDK projects, including the Example Project. However, if you have changed the default networking option for this project, you need to reset it to Spatial networking.</br>
To check which networking your project is using, look in its`DefaultGame.ini` configuration file (located in the `<ProjectRoot>\Game\Config` directory). If there is a`SpatialNetworking` option in the file, set it to `True` to enable Spatial networking. 
If there is no `SpatialNetworking` option <WHAT DO I DO?>.
```

Reply:
Just pushed another change based on your suggestion above. Let me know what you think.